### DISCUSSION TITLE
New installation the link on the home page at 8181 not workling

### DISCUSSION THREAD

**Discussion by izzywdev** [fonoster/yaps#623] [Q&A] [answered]
/api/auth/error returns Internal Server Error

Setup is on GCP VM. VPC firewall forwards 8181 to the machine.

NEXTAUTH_URL=http://55.55.55.55:8181
WEBUI_APP_URL=http://55.55.55.55:8181

and 

WEBUI_TEST_PHONE_SERVER=ws://55.55.55.55:5062

are the only lines I modified in the default .env file.

What am I missing?

Thanks a lot.

**Comment by teaminverso1** [comment]
I think you need to add the GitHub client ID, secret, and authorization callback URL. 

I could be wrong though as I'm getting the same error and cannot for the life of me figure out how to get Fonoster working.

**Comment by burnedsyn** [comment]
Hello is there any way to figure this out ? i have the same problem, i'm fairly new to this and as i see where to get and set id and secret i don't see the call back url in env  i have been through the doc but i don't see anything about this.

**Comment by psanders** [comment, marked as answer]
Sorry for the confusion.

We don't yet have a WebUI (it is being rebuilt).

I updated the self-hosting docs accordingly.

You can follow the progress around this topic [here.](https://github.com/orgs/fonoster/discussions/646)
